Neos.ContentRepository/Classes/Domain/Model/Workspace.php

```diff
@@ -33,7 +33,7 @@ class Workspace
     /**
      * This prefix determines if a given workspace (name) is a user workspace.
      */
-    const PERSONAL_WORKSPACE_PREFIX = 'user-';
+    public const PERSONAL_WORKSPACE_PREFIX = 'user-';
```

Comment:
Is it used elsewhere? If not, make it protected (or even private)?

Reply:
It is, in `Neos\ContentRepository\Domain\Model\Workspace\User`